src/core_functions/aggregate/distributive/arg_min_max.cpp

```diff
@@ -32,9 +32,15 @@ struct ArgMinMaxStateBase {
 	}
 
 	bool is_initialized;
+	bool arg_null;
 };
 
 // Out-of-line specialisations
+template <>
+void ArgMinMaxStateBase::CreateValue(string_t &value) {
+	new (&value) string_t(uint32_t(0));
```

Comment:
Sure! I only did this because assignment is more complex. (Also, I once helped a project get rid of huge amounts of code back in the day by converting their C++ object pager to use placement new, but I understand it can seem tasteless at times!)